directives/_static/js/aplus_tabs.js

```diff
@@ -0,0 +1,45 @@
+$(function() {
+
+    $('div.content-tabs').each(function() {
+        var contenttab_sel = $('<ul />', { class: "contenttab-selector" });
```

Comment:
Our new JS style guide uses two spaces for indentation: https://apluslms.github.io/contribute/styleguides/

Reply:
`contenttab_sel` is actually a new element, not a selector ("sel" is abbreviated from "selector"). I'm also confused of the CSS class name "contenttab-selector". Why is the word "selector" used there? Does the name describe the purpose of the element?

Reply:
I am referring to the tab itself and not the content. So basically if I have a group of tabs, each tab is a selector for the content. But, I can make the names semantically more correct.